### PR TITLE
Doc add tenant-federation in authentication-and-authorization

### PR DESCRIPTION
**Pull request by kehao95**
#### What this PR does
Add how `tenant-federation` works in Doc's **authentication and authorization**

#### Which issue(s) this PR fixes or relates to
https://github.com/grafana/mimir/discussions/1635

#### Checklist

- [ ] Tests updated
- [x] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/sources/operators-guide/securing/authentication-and-authorization.md
Comment (56quarters) [inline docs/sources/operators-guide/securing/authentication-and-authorization.md:12]: Could you add an example of what supplying mutiple tenant IDs looks like? E.g. `tenant-1|tenant-2|tenant-3`

**Review by 56quarters — COMMENTED** [review]
This looks good to me but I'll let our docs people weigh in. Thanks!

File: docs/sources/operators-guide/securing/authentication-and-authorization.md
Comment (56quarters) [inline docs/sources/operators-guide/securing/authentication-and-authorization.md:12]:
```suggestion
Queries can also be federated across multiple tenants when `tenant-federation` is enabled, refer to [Configuration Parameters]({{< relref "../configuring/reference-configuration-parameters/index.md#configuration-parameters" >}}). The tenant IDs involved need to be specified separated by a `|` character in the 'X-Scope-OrgID' header (e.g. `X-Scope-OrgID: tenant-1|tenant-2|tenant-3`).
```

Comment (kehao95) [reply docs/sources/operators-guide/securing/authentication-and-authorization.md]: Thank you, I've amend the commit.

File: docs/sources/operators-guide/securing/authentication-and-authorization.md
Comment (pracucci) [inline docs/sources/operators-guide/securing/authentication-and-authorization.md:12]: [nit] I would be more direct about how to enable tenant federation.

```suggestion
Queries can also be federated across multiple tenants when tenant federation is enabled setting `-tenant-federation.enabled=true`.
The tenant IDs involved need to be specified separated by a `|` character in the 'X-Scope-OrgID' header (e.g. `X-Scope-OrgID: tenant-1|tenant-2|tenant-3`).
```

**Review by pracucci — APPROVED** [review]
Thanks! LGTM (modulo a nit)

File: docs/sources/operators-guide/securing/authentication-and-authorization.md
Comment (osg-grafana) [inline docs/sources/operators-guide/securing/authentication-and-authorization.md:11]:
```suggestion
The query takes the tenant ID from the `X-Scope-OrgID` parameter that exists in the HTTP header of each request, for example `X-Scope-OrgID: tenant-id`.
```
Adhering to style to spell out abbreviations or shortenings.

File: docs/sources/operators-guide/securing/authentication-and-authorization.md
Comment (osg-grafana) [reply docs/sources/operators-guide/securing/authentication-and-authorization.md:12]: > [nit] I would be more direct about how to enable tenant federation.

```suggestion
You can federate queries across multiple tenants by using `true` in `-tenant-federation.enabled=true`. When you specify tenant IDs, separate them with a pipe (`|`) character in the 'X-Scope-OrgID' header, as in the example `X-Scope-OrgID: tenant-1|tenant-2|tenant-3`.
```

**Review by osg-grafana — CHANGES_REQUESTED** [review]
Change verbiage to speak more directly to the reader.

**Pull request merged**
